wgpu/_classes.py

```diff
@@ -380,16 +382,16 @@ def is_fallback_adapter(self):
     # IDL: [SameObject] readonly attribute GPUAdapterInfo info;
     @property
     def info(self):
-        """A dict with information about this adapter, such as the vendor and devicen name."""
+        """A dict with information about this adapter, such as the vendor and device name."""
         # Note: returns a dict rather than an GPUAdapterInfo instance.
         return self._adapter_info
 
     @apidiff.add("Useful in multi-gpu environments")
     @property
     def summary(self):
-        """A one-line summary of the info of this adapter (name, adapter_type, backend_type)."""
+        """A one-line summary of the info of this adapter (description, adapter_type, backend_type)."""
         d = self._adapter_info
-        return f"{d['device']} ({d['adapter_type']}) via {d['backend_type']}"
+        return f"{d['description']} ({d['adapter_type']}) via {d['backend_type']}"
```

Comment:
This was needed because of a mistake in wgpu-native. This was fixed in https://github.com/gfx-rs/wgpu-native/pull/418. We should change it back to `device` when we update wgpu-native :) I will either do that in this pr or directly after in a new pr.